### DISCUSSION TITLE
RE: GPL Violation by SMLight

### DISCUSSION THREAD

**Discussion by pykpkg47** [xyzroe/XZG#137] [General]
SMLight is using code from the xyzroe/ZigStarGW-FW (the predecessor of XZG) for its 2.x version firmware, which is GPL licensed. However, they are refusing to post the source code (https://github.com/smlight-dev/slzb-06-manual/issues/10). 

I started this discussion to coordinate an action to make this issue more public within the Smart Home community and to apply pressure on SMLight to comply with the GPL license.

Multiple open-source Smart Home projects, such as Home Assistant and Zigbee2MQTT, are recommending this manufacturer. However, I believe that open-source projects should not support a GPL violator. I plan to submit pull requests to these repositories to remove the recommendation until SMLight addresses this violation.

**Comment by xyzroe** [comment]
Nice plan. I will support you. Thanks for your action

**Reply by xyzroe** [reply]
@pykpkg47 any news? maybe I need to do something?
